app/assets/stylesheets/components/_message_show.scss

```diff
@@ -41,4 +41,37 @@ img {
 
 }
 
+.form-control {
```

Comment:
This will also apply to ALL `.form-control` elements on the website. This is the default class for all simple forms so probably on a lot of pages...